hbase-http/src/main/java/org/apache/hadoop/hbase/http/ProfileOutputServlet.java

```diff
@@ -37,6 +38,8 @@
   private static final long serialVersionUID = 1L;
   private static final Logger LOG = LoggerFactory.getLogger(ProfileOutputServlet.class);
   private static final int REFRESH_PERIOD = 2;
+  // Alphanumeric characters, plus percent (url-encoding), equals, and ampersand
+  private static final Pattern ALPHA_NUMERIC = Pattern.compile("[a-zA-Z0-9\\%\\=\\&]*");
```

Comment:
nit:should also accept a literal `+` for spaces I think?

Reply:
Agree!